From c70029539d0aec5df0c1e4203359335a3841a1e5 Mon Sep 17 00:00:00 2001
From: Andreas Rammhold <andreas@rammhold.de>
Date: Thu, 9 May 2019 11:15:22 +0200
Subject: [PATCH 14/18] add rootprefix to lookup dir paths

systemd does not longer use the UDEVLIBEXEC directory as root for
discovery default udev rules. By adding `$out/lib` to the lookup paths
we should again be able to discover the udev rules amongst other default
files that I might have missed.
---
 src/basic/def.h | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/basic/def.h b/src/basic/def.h
index 970654a1ad..bb261040f8 100644
--- a/src/basic/def.h
+++ b/src/basic/def.h
@@ -39,13 +39,15 @@
         "/run/" n "\0"                          \
         "/usr/local/lib/" n "\0"                \
         "/usr/lib/" n "\0"                      \
-        _CONF_PATHS_SPLIT_USR_NULSTR(n)
+        _CONF_PATHS_SPLIT_USR_NULSTR(n)         \
+        ROOTPREFIX "/lib/" n "\0"
 
 #define CONF_PATHS_USR(n)                       \
         "/etc/" n,                              \
         "/run/" n,                              \
         "/usr/local/lib/" n,                    \
-        "/usr/lib/" n
+        "/usr/lib/" n,                          \
+        ROOTPREFIX "/lib/" n
 
 #define CONF_PATHS(n)                           \
         CONF_PATHS_USR(n)                       \
-- 
2.26.2

